draft-ZIP-0226.rst

```diff
@@ -0,0 +1,303 @@
+::
+
+  ZIP: 226
+  Title: Transfer and Burn of Zcash Shielded Assets
+  Owners: Daniel Benarroch <daniel@qed-it.com>
+          Pablo Kogan <pablo@qed-it.com>
+          Aurelien Nicolas <aurel@qed-it.com>
+  Credits: Daira Hopwood
+           Jack Grigg
+           Shahaf Nacshon
+           Vivek Arte
+  Status: Draft
+  Category: Consensus
+  Created: 2022-05-01
+  License: MIT
+  Discussions-To: <https://github.com/zcash/zips/issues/618>
+
+
+Terminology
+===========
+
+The key word "MUST" in this document is to be interpreted as described in RFC 2119 [#RFC2119]_.
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200 [#zip-0200]_.
+
+The term "Orchard" in this document is to be interpreted as described in ZIP 224 [#zip-0224]_.
+
+We define the following additional terms:
+
+- Asset: A type of note that can be transferred on the Zcash block chain. ZEC is the default (and currently the only defined) asset for the Zcash mainnet.
+- Zcash Shielded Asset: an asset with issuance defined on the Zcash block chain, and, for now, belonging to the Orchard anonymity pool.
+- Wrapped Asset: a ZSA asset with native issuance defined outside the Zcash block chain.
```

Comment:
Can we define native issuance, and distinguish from "on chain" issuance above?

Reply:
Done.